tests/bootstrap.php

```diff
@@ -47,3 +47,5 @@
     ],
 ]);
 Configure::write('debug', true);
+
+Cache::clearAll();
```

Comment:
Is this necessary? Cache should be already disabled for all tests, and explicitly enabled for test cases that perform assertions on cache. Assertions on cache, however, should use an in-memory cache engine. If it's not the case, I'd solve this issue instead of clearing the whole cache.

Reply:
Cache seems to be disabled only for `_bedita_object_types_`.
`_cake_core_` and `_cake_model_` instead are written and used on testsuite launch
one solution is to clear only those two - but this could be done of course in a different PR/issue